### PR TITLE
Version hidden by commenting the labels 

### PR DESCRIPTION
**Pull request by Rajrits**
for #759

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -161,11 +161,11 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
...
-            {label: '1.3.2', to: '/docs/1.3.2/about/introduction'},
-            {label: '1.3.1', to: '/docs/1.3.1/about/introduction'},
-            {label: '1.3.0', to: '/docs/1.3.0/introduction'},
-            {label: '1.2.0', to: '/docs/1.2.0/introduction'},
+            // {label: '1.4.0', to: '/docs/latest/about/introduction'},
+            // {label: '1.3.2', to: '/docs/1.3.2/about/introduction'},
+            // {label: '1.3.1', to: '/docs/1.3.1/about/introduction'},
+            // {label: '1.3.0', to: '/docs/1.3.0/introduction'},
+            // {label: '1.2.0', to: '/docs/1.2.0/introduction'},
             {label: '1.1.1', to: '/docs/1.1.1/introduction'},
```
Comment (casionone) [inline docusaurus.config.js]: The latest two versions（1.4.0 and 1.3.2） are retained
It is also necessary to update the overview information here(https://deploy-preview-761--apache-linkis-website.netlify.app/versions) at the same time (all document versions need to be listed here) 
image

Comment (casionone) [inline docusaurus.config.js]: Need to retain
- 1.4.0  
- 1.3.2 
- Next(1.5.0)
- All version

**Comment by casionone** [comment]
image
image

File: docusaurus.config.js (hunk 1 above)
Comment (Rajrits) [reply docusaurus.config.js]: I've done it, Please review it again sir.

**Pull request merged**